### PR TITLE
Fix typos and misc in distribution functions

### PR DESCRIPTION
**Pull request by talgalili**
Test plan: None.

```
== NO RELEASE NOTE ==
```

### CODE REVIEW

File: presto-main/src/main/java/com/facebook/presto/operator/scalar/MathFunctions.java [hunk 1]
```diff
@@ -849,7 +849,7 @@ public static double betaCdf(
         return distribution.cumulativeProbability(value);
     }
 
-    @Description("Inverse of Cauchy cdf for a given probability, median, and scale (gamma)")
+    @Description("Inverse of Cauchy cdf for a given probability, median, and scale (gamma) parameters")
```
Comment (rschlussel) [inline presto-main/src/main/java/com/facebook/presto/operator/scalar/MathFunctions.java:852]: I don't think "parameters" is needed at the end here

Comment (talgalili) [reply presto-main/src/main/java/com/facebook/presto/operator/scalar/MathFunctions.java]: Thanks @rschlussel - I've removed "parameters".

File: presto-main/src/main/java/com/facebook/presto/operator/scalar/MathFunctions.java [hunk 2]
```diff
@@ -863,7 +863,7 @@ public static double inverseCauchyCdf(
         return distribution.inverseCumulativeProbability(p);
     }
 
-    @Description("Cauchy cdf for a given value, median, and scale (gamma)")
+    @Description("Cauchy cdf for a given value, median, and scale (gamma) parameters")
```
[the remark this answers is not in the document]
Comment (talgalili) [reply presto-main/src/main/java/com/facebook/presto/operator/scalar/MathFunctions.java]: Thanks @rschlussel - ditto :)

**Comment by rschlussel** [comment]
Thanks! I'll merge once the tests finish

**Review by jainxrohit — APPROVED** [review]

**Comment by talgalili** [comment]
Thanks @rschlussel and @jainxrohit !

**Pull request merged**